test/integration/custom-routes/test/index.test.js

```diff
@@ -71,18 +71,28 @@ const runTests = (isDev = false) => {
     expect(pathname).toBe('/')
   })
 
+  it('should server static files through a rewrite', async () => {
+    const text = await renderViaHTTP(appPort, '/hello-world')
+    expect(text).toBe('hello world!')
+  })
+
   it('should rewrite with params successfully', async () => {
     const html = await renderViaHTTP(appPort, '/test/hello')
     expect(html).toMatch(/Hello/)
   })
 
-  it('should overwrite param values correctly', async () => {
-    const html = await renderViaHTTP(appPort, '/test-overwrite/first/second')
-    expect(html).toMatch(/this-should-be-the-value/)
-    expect(html).not.toMatch(/first/)
-    expect(html).toMatch(/second/)
+  it('should double redirect successfully', async () => {
+    const html = await renderViaHTTP(appPort, '/docs/github')
+    expect(html).toMatch(/hi there/)
   })
 
+  // it('should overwrite param values correctly', async () => {
+  //   const html = await renderViaHTTP(appPort, '/test-overwrite/first/second')
+  //   expect(html).toMatch(/this-should-be-the-value/)
+  //   expect(html).not.toMatch(/first/)
+  //   expect(html).toMatch(/second/)
+  // })
```

Comment:
Do we not want initial params to be passed down? This is what it was mainly testing